src/styles/searchResults.css

```diff
@@ -0,0 +1,30 @@
+.masonry-grid {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 30px;
+  width: 100%;
```

Comment:
The justify-content and width properties arent doing anything, these should be removed